packages/paste-website/src/component-examples/filter-group/Examples.tsx

```diff
@@ -1,15 +1,22 @@
 import * as React from 'react';
+import {Anchor} from '@twilio-paste/anchor';
 import {Box} from '@twilio-paste/box';
 import {DefaultFilterGroup} from './components/DefaultFilterGroup';
 
 export const DefaultFilterGroupExample: React.FC = () => (
   <Box marginBottom="space70" data-cy="filter-group-default-example">
     <DefaultFilterGroup />
+    <Anchor showExternal href="https://codesandbox.io/s/default-filter-group-rloskb?file=/src/App.tsx">
```

Comment:
suggestion: I feel like the link gets lost a little, would it be possible to make these `button as="a"`s instead of anchors?

Reply:
I think if we've built button correctly, we will tell you not to do that, and just use an Anchor. I even think we throw an error.

If you want to increase the hierarchy, consider just bolding the text in side the Anchor, with `<strong />` or wrap in `<Text />` and adjust the `fontWeight`

Reply:
Actually, I think I understand what you mean. A small size button might be OK, but generally speaking we provide accessibility guidance to only use links that look like buttons, [sparingly](https://deploy-preview-2320--paste-docs.netlify.app/components/button#accessibility)

Reply:
I had the [Button with Link Functionality](https://paste.twilio.design/components/button#buttons-with-link-functionality) in mind

Reply:
Yeah... I think I would rather make it bold than make it look like a button. Is that ok with y'all?